modules/claims_api/app/sidekiq/claims_api/report_unsuccessful_submissions.rb

```diff
@@ -26,123 +24,5 @@ def perform
-    end
-
-    def claims_totals
-      @claims_consumers.map do |cid|
-        counts = ClaimsApi::AutoEstablishedClaim.where(created_at: @from..@to, cid:).group(:status).count
-        totals = counts.sum { |_k, v| v }.to_f
-
-        percentage_with_flashes = "#{((with_flashes(cid:) / totals) * 100).round(2)}%"
-        percentage_with_special_issues = "#{((with_special_issues(cid:) / totals) * 100).round(2)}%"
-
-        if totals.positive?
-          consumer_name = ClaimsApi::CidMapper.new(cid:).name
-          {
-            consumer_name => counts.merge(totals:,
-                                          percentage_with_flashes: percentage_with_flashes.to_s,
-                                          percentage_with_special_issues: percentage_with_special_issues.to_s)
-                                   .deep_symbolize_keys
-          }
-        end
-      end
-    end
-
-    def poa_totals
-      @poa_consumers.map do |cid|
-        counts = ClaimsApi::PowerOfAttorney.where(created_at: @from..@to, cid:).group(:status).count
-        totals = counts.sum { |_k, v| v }
-
-        if totals.positive?
-          consumer_name = ClaimsApi::CidMapper.new(cid:).name
-          {
-            consumer_name => counts.merge(totals:)
-                                   .deep_symbolize_keys
-          }
-        end
-      end
-    end
-
-    def unsuccessful_poa_submissions
-      errored_poas.pluck(:cid, :created_at, :id).map do |cid, created_at, id|
-        { id:, created_at:, cid: }
-      end
-    end
-
-    def errored_poas
-      ClaimsApi::PowerOfAttorney.where(
-        created_at: @from..@to,
-        status: %w[errored]
-      ).order(:cid, :status)
-    end
-
-    def itf_totals
-      @itf_consumers.map do |cid|
-        counts = ClaimsApi::IntentToFile.where(created_at: @from..@to, cid:).group(:status).count
-        totals = counts.sum { |_k, v| v }
-
-        if totals.positive?
-          consumer_name = ClaimsApi::CidMapper.new(cid:).name
-          {
-            consumer_name => counts.merge(totals:)
-                                   .deep_symbolize_keys
-          }
-        end
-      end
-    end
-
-    def ews_totals
-      @ews_consumers.map do |cid|
-        counts = ClaimsApi::EvidenceWaiverSubmission.where(created_at: @from..@to, cid:).group(:status).count
-        totals = counts.sum { |_k, v| v }
-
-        if totals.positive?
-          consumer_name = ClaimsApi::CidMapper.new(cid:).name
-          {
-            consumer_name => counts.merge(totals:)
-                                   .deep_symbolize_keys
-          }
-        end
-      end
-    end
-
-    def unsuccessful_evidence_waiver_submissions
-      errored_ews.pluck(:cid, :created_at, :id).map do |cid, created_at, id|
-        { id:, created_at:, cid: }
-      end
-    end
-
-    def errored_ews
-      ClaimsApi::EvidenceWaiverSubmission.where(
-        created_at: @from..@to,
-        status: %w[errored]
-      ).order(:cid, :status)
-    end
   end
```

Comment:
Quite a few of the line changes stem from this refactor which was in an effort to share the methods being used by both reporting jobs. -117 here and then +125 in the shared `reporting_base` file.  The only reason I would not make this change would be to avoid the line count warning, however since these _**are**_ shared methods I believe this change is worth it.